packages/connect-explorer/src/actions/trezorConnectActions.ts

```diff
@@ -19,6 +23,14 @@ export function onSelectDevice(path: string) {
     };
 }
 
+export const onConnectOptionChange = (option: string, value: any) => ({
+    type: ACTIONS.ON_CHANGE_CONNECT_OPTION,
+    payload: {
+        option,
+        value,
+    },
+});
+
```

Comment:
yes, this is something we want in general. in the future we would like to have some UI presenting information about options for TrezorConnect.init. So far, lets use it for our need in tests only (incomplete and hidden), once we are finished, we can show it publicly.

Reply:
Yes, I agree we can add more UI sugar :candy:  once we add more options and probably documentation and so on.